examples/dreambooth/train_dreambooth_lora.py

```diff
@@ -1374,7 +1374,7 @@ def compute_text_embeddings(prompt):
         pipeline = pipeline.to(accelerator.device)
 
         # load attention processors
-        pipeline.load_lora_weights(args.output_dir, weight_name="pytorch_lora_weights.bin")
+        pipeline.load_lora_weights(args.output_dir, weight_name="pytorch_lora_weights.safetensors")
```

Comment:
Does `save_lora_weights()` have `safe_serialization` set to True default? Otherwise, this will fail.

Reply:
Also, we don't need to explicitly specify the `weight_name` actually since the naming is in the `diffusers` format.

Reply:
`safe_serialization` defaults to `True` in the `LoraLoaderMixin`
https://github.com/huggingface/diffusers/blob/1dd34410cc9204d14c9cefd7149d3064d0e67727/src/diffusers/loaders.py#L1418

Reply:
Regarding, `weight_name`. Not setting it while saving is fine, however, loading looks like it needs a path to the weights
https://github.com/huggingface/diffusers/blob/1dd34410cc9204d14c9cefd7149d3064d0e67727/src/diffusers/loaders.py#L906

Reply:
Yeah, you only need `pipeline.load_lora_weights(args.output_dir)`.